app/pages/project/project-page.jsx

```diff
@@ -0,0 +1,173 @@
+import Translate from 'react-translate-component';
+import { Markdown } from 'markdownz';
+import { sugarClient } from 'panoptes-client/lib/sugar';
+import ProjectNavbar from './project-navbar';
+import PotentialFieldGuide from './potential-field-guide';
+
+export default class ProjectPage extends React.Component {
+  constructor() {
+    super();
+    this.lastSugarSubscribedID = null;
+  }
+
+  getChildContext() {
+    return this.context.geordi;
+  }
+
+  componentDidMount() {
+    document.documentElement.classList.add('on-project-page');
+    this.updateSugarSubscription(this.props.project);
+    this.context.geordi && this.context.geordi.remember({ projectToken: this.props.project.slug });
+  }
+
+  componentWillReceiveProps(nextProps) {
+    if (nextProps.project !== this.props.project) {
+      this.updateSugarSubscription(nextProps.project);
+      this.context.geordi && this.context.geordi.remember({ projectToken: nextProps.project.slug });
+    }
+  }
+
+  componentWillUnmount() {
+    document.documentElement.classList.remove('on-project-page');
+    this.updateSugarSubscription(null);
+    this.context.geordi.forget(['projectToken']);
+  }
+
+  updateSugarSubscription(project) {
+    if (this.lastSugarSubscribedID) {
+      sugarClient.unsubscribeFrom(`project-${this.lastSugarSubscribedID}`);
+    }
+    if (project) {
+      sugarClient.subscribeTo(`project-${this.lastSugarSubscribedID}`);
+    }
+  }
+
+  render() {
+    const projectPath = `/projects/${this.props.project.slug}`;
+    const onHomePage = this.props.routes[2].path === undefined;
+    const pages = this.props.pages.reduce((map, page) => {
+      map[page.url_key] = page;
+      return map;
+    }, {});
+
+    const announcement = (this.props.project.configuration && this.props.project.configuration.announcement) ?
+      <div className="informational project-announcement-banner">
+        <Markdown>
+          {this.props.project.configuration.announcement}
+        </Markdown>
+      </div> :
+      null;
+
+    const navBar = (!onHomePage) ?
+      <div>
+        <ProjectNavbar {...this.props} />
+        {announcement}
+      </div> :
+      null;
+
+    const launchApproved = (!this.props.project.launch_approved) ?
+      <Translate
+        component="p"
+        className="project-disclaimer"
+        content="project.disclaimer"
+      /> :
+      null;
+
+    const potentialFieldGuide = (this.props.location.pathname !== projectPath) ?
+      <PotentialFieldGuide
+        guide={this.props.guide}
+        guideIcons={this.props.guideIcons}
+      /> :
+      null;
+
+    let backgroundStyle = {};
+    if (this.props.background) {
+      backgroundStyle = {
+        backgroundImage: `radial-gradient(rgba(0, 0, 0, 0.2), rgba(0, 0, 0, 0.8)), url('${this.props.background.src}')`
+      };
+    }
+
+    return (
+      <div className="project-page project-background" style={backgroundStyle}>
+        {navBar}
+        {launchApproved}
+        {potentialFieldGuide}
```

Comment:
I think these two lines were after the children in the original code.

Reply:
Good catch, thank you!